### PR TITLE
Making template names consistent

### PR DESCRIPTION
**Pull request by mathewc**
Playing around in the UI, the inconsistencies are bothering me. Addressing some of them :)

### CODE REVIEW

File: Templates/QueueTrigger-Php/metadata.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
     "defaultFunctionName": "QueueTriggerPhp",
     "description": "A PHP function that will be run whenever a message is added to a specified Azure Storage queue",
-    "name": "QueueTrigger Php Function",
+    "name": "QueueTrigger - Php",
```
Comment (davidebbo) [inline Templates/QueueTrigger-Php/metadata.json]: Since you're in there, PHP is always upper case. Couple places in here.

**Comment by christopheranderson** [comment]
Looks fine. I like the shorter names. I think this fits our beta milestone bar. :+1:

**Comment by davidebbo** [comment]
:shipit: I've been wanting to do the same thing. Getting rid of 'function' makes the name shorter.

File: Templates/QueueTrigger-Php/metadata.json (hunk 1 above)
Comment (mathewc) [inline Templates/QueueTrigger-Php/metadata.json]: done

**Pull request closed without merging**